activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/resources/application.properties

```diff
@@ -18,3 +18,5 @@ keycloak.security-constraints[0].securityCollections[0].patterns[0]=/v1/*
 keycloak.security-constraints[1].authRoles[0]=ACTIVITI_ADMIN
 keycloak.security-constraints[1].securityCollections[0].patterns[0]=/admin/*
 
+activiti.cloud.audit.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service

Reply:
we removed these properties